test/system/src/test/java/io/pravega/test/system/PravegaTest.java

```diff
@@ -142,6 +147,8 @@ public void simpleTest() throws InterruptedException {
             writer.flush();
             Thread.sleep(500);
         }
+        WriteCount.set(NUM_EVENTS); /* Unconditionally setting write count here as writeEvent was not returning the write status */
+        writer.close();
```

Comment:
writer is closed as part of @Cleanup.

Reply:
Ok, I will remove the newly added write close and update the patch accordingly

Reply:
Yes the lombok annotation@Cleanup invokes close() in the finally block.

Reply:
ok, removed the explicit close()